File: tests/fib.lua
function fibR(n)

    if (n < 2) then return n end
    return (fibR(n-2) + fibR(n-1))
end

local start = os.clock()
N = 34 --Should return 433494437
print("fib: " .. fibR(N))
io.write(string.format("elapsed: %.8f\n", os.clock() - start))

File: tests/fib.c
#include <stdio.h>

int fibR(int n)
{
    if (n < 2) return n;
    return (fibR(n-2) + fibR(n-1));
}

int main()
{
    int N;
    N = 34; //Should return 433494437
    printf("fib: %d\n", fibR(N));
    return 0;
}
